src/main/webapp/resources/js/pages/sequence-files/components/FastQC.jsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * This file is responsible for displaying the
+ * tabs required and default renders the
+ * FastQCCharts component.
+ */
+
+import React, { Suspense, useEffect, useState } from "react";
+import { setBaseUrl } from "../../../utilities/url-utilities";
+import { PageWrapper } from "../../../components/page/PageWrapper";
+import { Link, Location, Router } from "@reach/router";
+import { Badge, Menu } from "antd";
+import { ContentLoading } from "../../../components/loader";
+
+import { SPACE_MD, SPACE_XS } from "../../../styles/spacing"
+import { getFastQCDetails } from "../../../apis/files/sequence-files";
+import {
+  seqObjId,
+  seqFileId,
+  urlMatch
+} from "../fastqc-constants";
+import { InfoAlert } from "../../../components/alerts";
+import { blue6 } from "../../../styles/colors";
+
+const FastQCDetails = React.lazy(() => import("./FastQCDetails"));
+const FastQCCharts = React.lazy(() => import("./FastQCCharts"));
+const OverRepresentedSequences = React.lazy(() => import("./OverRepresentedSequences"));
+
+export default function FastQC() {
+  const [loading, setLoading] = useState(true);
+  const [fastQC, setFastQC] = useState({});
+  const [file, setFile] = useState({});
+
+  const DEFAULT_URL = setBaseUrl(urlMatch[0]);
+
+  const [selectedKeys, setSelectedKeys] = React.useState(() => {
+    // Create a regex from the string with an additional string at the end
+    const regexStr = urlMatch[0] + "/(\\w*)?";
+    const urlRegex = new RegExp(regexStr, "i");
+
+    const urlMatchRegex = window.location.href.match(urlRegex);
+    // Grab the key if it exists from the match array otherwise default to 'charts'
+    return urlMatchRegex !== null ? urlMatchRegex[1] : "charts";
+  });
+
+  useEffect(() => {
+    getFastQCDetails(seqObjId, seqFileId).then(({ analysisFastQC, sequenceFile }) => {
+      setFile(sequenceFile);
+      setFastQC(analysisFastQC);
+      setLoading(false);
+    });
+  }, []);
+
+  return (
+    <PageWrapper title={file.fileName}>
```

Comment:
Make sure to set up the onBack handler which should take you back to the sample files page.

Reply:
As discussed we are going to leave this for now